controllers/apply-next/simple/form.js

```diff
@@ -0,0 +1,1270 @@
+                en: `It's important to make sure the date of birth is correct, as errors will fail our authenticity checks and delay your application.`,
+                cy: ''
+            }),
+            type: 'date',
+            attributes: {
+                max: moment()
+                    .subtract(minAge, 'years')
+                    .format('YYYY-MM-DD')
+            },
+            isRequired: true,
+            schema: Joi.when(Joi.ref('organisation-type'), {
+                is: organisationTypes.school.value,
+                then: Joi.any().optional(),
+                otherwise: commonValidators.dateOfBirth(minAge).required()
+            }),
+            messages: [
+                {
+                    type: 'base',
+                    message: { en: 'Enter a date of birth', cy: '' }
+                },
+                {
+                    type: 'any.invalid',
+                    message: { en: 'Enter a real date', cy: '' }
+                },
+                {
+                    type: 'dateParts.dob',
+                    message: { en: `Must be at least ${minAge} years old`, cy: '' }
+                }
+            ]
+        };
+
+        return { ...defaultProps, ...props };
+    }
+
+    function communicationNeedsField(props) {
+        const defaultProps = {
+            type: 'radio',
+            options: [
+                { value: 'audiotape', label: localise({ en: 'Audiotape', cy: '' }) },
+                { value: 'braille', label: localise({ en: 'Braille', cy: '' }) },
+                { value: 'disk', label: localise({ en: 'Disk', cy: '' }) },
+                { value: 'large-print', label: localise({ en: 'Large print', cy: '' }) },
+                { value: 'letter', label: localise({ en: 'Letter', cy: '' }) },
+                { value: 'sign-language', label: localise({ en: 'Sign language', cy: '' }) },
+                { value: 'text-relay', label: localise({ en: 'Text relay', cy: '' }) }
+            ],
+            get schema() {
+                return Joi.array()
+                    .items(Joi.string().valid(this.options.map(option => option.value)))
+                    .single()
+                    .optional();
+            },
+            messages: [
+                {
+                    type: 'any.allowOnly',
+                    message: { en: 'Choose from one of the options provided', cy: '' }
+                }
+            ]
+        };
+
+        return { ...defaultProps, ...props };
+    }
+
+    const allFields = {
+        projectName: {
+            name: 'project-name',
+            label: localise({ en: 'What is the name of your project?', cy: '' }),
+            explanation: localise({ en: 'The project name should be simple and to the point', cy: '' }),
+            type: 'text',
+            isRequired: true,
+            schema: Joi.string().required(),
+            messages: [{ type: 'base', message: { en: 'Enter a project name', cy: '' } }]
```

Comment:
Messages are the one place where we are not (currently) translating the strings inline as this happens in a different module and the changes here were already substantial enough. I think it would be natural to update these to behave the same way as other strings in this file.